docs/content/guide/decorators.ngdoc

```diff
@@ -0,0 +1,356 @@
+@ngdoc overview
+@name Decorators
+@sortOrder 345
+@description
+
+# Decorators in AngularJS
+
+<div class="alert alert-warning">
+  **NOTE:** This guide is targeted towards developers who are already familiar with AngularJS basics.
+  If you're just getting started, we recommend the {@link tutorial/ tutorial} first.
+</div>
+
+## What are decorators?
+
+Decorators are used to facilitate a decorator design pattern. This pattern is used to separate modification or
+*decoration* of a class without modifying the original source code. In Angular, decorators are functions that allow a
+service to be modified after their instantiation.
```

Comment:
OK, for services, you first have to instantiate them in order to decorate them, but the current phrasing might give the impression that a service could be instantiated and used in its original form and later be decorated.